pkgs/tools/admin/mycli/default.nix

```
@@ -38,5 +36,6 @@ buildPythonApplication rec {
     '';
     homepage = http://mycli.net;
     license = lib.licenses.bsd3;
+    maintainers = [ lib.maintainers.jojosch ];
```

in case others want to join in on the fun.
```suggestion
    maintainers = with lib.maintainers; [ jojosch ];
```

---

but, they can add this in the other PR.